src/pages/collaborate.tsx

```diff
@@ -12,4 +12,4 @@ const Collaborate = () => {
   return null;
 };
 
-export default Collaborate;
\ No newline at end of file
+export default Collaborate;
```

Comment:
I was running into an error that told me to add a space here. So thats what this is for 😄